### PR TITLE
Fixes #9682 - added suport for i18n

### PR DESCRIPTION
**Pull request by mbacovsky**
- there is new resource on transifex: https://www.transifex.com/projects/p/foreman/resource/hammer_cli_foreman_docker/ and initial strings were uploaded there
- docs on how to add new langs from transifex in `locale/README.md`
- with this patch `LANG=<lang> hammer docker -h` should work for lang that has translations available. With debug mode (-d) you can see that the locales were loaded

### CODE REVIEW

**Comment by domcleal** [comment]
After merge, please update the raw GitHub URL to download the POT file at https://www.transifex.com/projects/p/foreman/resources/, click Auto update resources & paste it in.

**Comment by mbacovsky** [comment]
@domcleal will do, thanks for instructions!

File: .tx/config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://www.transifex.com
+
+[foreman.hammer-cli-foreman-docker]
+file_filter = locale/<lang>/hammer-cli-foreman-docker.po
```
Comment (domcleal) [inline .tx/config]: Your PO/POT files have underscores, but this uses hyphens.  For hammer..discovery, I think @orrabin stuck with underscores in the end.

**Comment by daviddavis** [comment]
ACK

**Pull request merged**